ynr/apps/candidates/templates/candidates/constituencies.html

```diff
@@ -30,6 +30,9 @@ <h3>
     {% elif postextraelection.suggestedpostlock_set.exists %}
      <abbr title="Someone suggested locking this post">🔓</abbr>
     {% endif %}
+    {% if postextraelection.cancelled %}
+      <abbr title="The poll for this election was cancelled">❌</abbr>
```

Comment:
Couple of things on this (and similar lines). I get that this is more of a comment on the way we're doing other stuff like this but…well, you gotta start somewhere.

1. Would it be possible to move this to a property on the model like `get_cancelled_html_message` or something?
2. Because cancelled elections are rare, I think it might be better to spell it out. I know we have a pattern of using icons on this page, but firstly they're not ideal on their own (I'd like to change this in future) and second they're used on basically every ballot paper, meaning that there is value in learning what they mean. Seeing an 'X' once in a list of ~3,000 items is less useful.

Reply:
Yep. That's a valid point that this is quite a rare condition. I'll have another think on Monday